client/pages/randomize.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Button from 'react-bootstrap/Button';
+import fetch from 'node-fetch';
+
+export default class Randomize extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedRestaurant: []
+    };
+    this.handleRandomizer = this.handleRandomizer.bind(this);
+  }
+
+  componentDidMount() {
+    fetch('/api/random')
+      .then(response => response.json())
+      .then(data => this.setState({
+        selectedRestaurant: data
+      }));
+  }
+
+  handleRandomizer(e) {
+    this.postDb();
+  }
+
+  postDb(restaurant) {
+    const requestOptions = {
+      method: 'PATCH',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify(restaurant)
+    };
+
+    fetch('/api/random', requestOptions)
+      .then(response => response.json());
+  }
```

Comment:
It's not really clear to me what this code is supposed to be doing. This component's only job should be to get a random restaurant and display it (like the figma screen showing the restaurant info).